roles/as_code/lib/reliability_engineer.rb

```diff
@@ -61,15 +59,23 @@
 
   salary 40_000..75_000
 
+  expected 'to be ambassadors of reliability engineering culture',
+    'We expect our Reliability Engineers to contribute to our reliability engineering culture, standards and thought leadership. We expect Reliability Engineers to champion existing standards by working closely with delivery teams to coach and mentor them. We also expect all Reliability Engineers to be involved in discussions around improving our standards.'
+
+  expected 'to proactively identify technical improvements and recommend them to delivery teams',
+    'We expect our Reliability Engineers to be able to spot areas for improvement in codebases such as test suites, dependency upgrades, performance issues detected by monitoring. We also expect them to be able to highlight these issues in a jargon free way and be able to convey the relative urgency of the issues ultimately leading to codebases being improved.'
+
   expected 'to autonomously deliver working software',
     'We expect our Reliability Engineers to be able to work on entire features, from conception through to production deployment, without guidance and direction from others. This doesn\'t mean they have to be delivered without pairing, or asking advice from the wider team. What it does mean is the primary force for ensuring the feature is built and deployed in a timely fashion is the responsiblity of the Reliability Engineer working on the problem.'
```

Comment:
Will we expect REs to work on features from conception to production as of January 2019?

My understanding of the New World was that the reduction of feature delivery by the SRE team was one of the primary goals.

Reply:
Great catch, will update!